README.md

```diff
@@ -128,7 +135,8 @@ just "Git Documentation":
 Once you are done testing the translation (see below), commit the
 documentation.XX.po files and ask the l10n coordinator to pull from you.
 
-Updating a documentation.XX.po file
+## Updating a documentation.XX.po file
```

Comment:
There's inline shell code and a diff hunk in this chapter that you missed. Those should probably be marked as code, too.